### PR TITLE
feat(storybook): add MembersSelector to storybook

### PR DESCRIPTION
**Pull request by osmaczko**
closes: #8178
fixes: #8210

### Affected areas

members selector, members editor

### Video

https://user-images.githubusercontent.com/33099791/201916638-8ebfbb66-233f-4c4b-b9dd-c5a3f1ac5a39.mp4

### CODE REVIEW

File: storybook/pages/MembersSelectorPage.qml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+            for(let i = 0; i < usersModel.count; i++) {
+                const obj = usersModel.get(i)
+                temporaryModel.append(obj)
+            }
+        }
+
+        function updateGroupMembers() {
+            let users = []
+            for(let i = 0; i < temporaryModel.count; i++) {
+                let obj = temporaryModel.get(i)
```
Comment (micieslak) [inline storybook/pages/MembersSelectorPage.qml:130]: const

File: storybook/pages/MembersSelectorPage.qml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+        function resetTemporaryModel() {
+            temporaryModel.clear()
+            for(let i = 0; i < usersModel.count; i++) {
+                const obj = usersModel.get(i)
+                temporaryModel.append(obj)
+            }
+        }
+
+        function updateGroupMembers() {
+            let users = []
```
Comment (micieslak) [inline storybook/pages/MembersSelectorPage.qml:128]: const

**Review by micieslak — APPROVED** [review]
Looks nice. Components are exposed with all their properties. It shows nicely what kind of problems we have here. The temporary model which is part of UI logic is implemented in backend. It causes that's necessary to reimplement logic related to that temporary model in js. Additionally `MembersSelectorView` creates temporary list internally but `MembersEditSelectorView` relies on `usersStore` from the backend. At some point we probably should remove ui related stuff from backend and handle it on ui side (directly in qml or using additional cpp component instantiated from qml).

Some minor comments, feel free to skip.

**Review by mprakhov — APPROVED** [review]
LGTM

**Comment by osmaczko** [comment]
> Looks nice. Components are exposed with all their properties. It shows nicely what kind of problems we have here. The temporary model which is part of UI logic is implemented in backend. It causes that's necessary to reimplement logic related to that temporary model in js. Additionally `MembersSelectorView` creates temporary list internally but `MembersEditSelectorView` relies on `usersStore` from the backend. At some point we probably should remove ui related stuff from backend and handle it on ui side (directly in qml or using additional cpp component instantiated from qml).
> 
> Some minor comments, feel free to skip.

Thanks for reviewing, these are good points. There are two ideas to solve it:
- create a temporary model entirely in qml and hide js imperative code under the abstraction
- create a temporary model entirely in C++ and instantiate it in qml

Nevertheless, it should not be a part of this PR.

**Pull request merged**